dso-l2/src/main/java/com/tc/server/TCServerMain.java

```diff
@@ -100,6 +100,11 @@ private static void writeVersion() {
       String longPatchString = info.toLongPatchString();
       LOGGER.info(longPatchString);
     }
+
+    String versionMessage = info.versionMessage();
+    if (versionMessage != null && versionMessage.length() > 0) {
```

Comment:
This will never be null by contract and I would replace the length check by `String.isEmpty()`